firstvoices/backend/utils/contact_us_utils.py

```diff
@@ -0,0 +1,33 @@
+import logging
+
+from backend.models import AppJson
+from backend.views.exceptions import ContactUsError
+
+
+def get_fallback_emails():
+    """
+    Returns the list of default emails to use if no emails or users are set on the site contact_users and
+    contact_emails fields. The list of default emails are set in the AppJson model with the key
+    'contact_us_default_emails'.
+    """
+
+    logger = logging.getLogger(__name__)
+    if AppJson.objects.filter(key="contact_us_default_emails").count() == 0:
+        logger.error(
+            'No default email set in AppJson model with key "contact_us_default_email".'
+        )
+        raise ContactUsError()
```

Comment:
Sorry, I must have missed this one-- this should probably be an ImproperlyConfigured exception with the message in the exception instead of in a log. Same for the one below